srcs/httpMethod/HTTPMethod.cpp

```diff
@@ -59,7 +59,36 @@ int do_put(std::string &response_message_str,
   // MUST

```

Comment:
the code is just restructured. The only difference is we use body << contents instead of newfile << contents
related to issue #69